js/kafka/kafka-client.js

```diff
@@ -38,6 +38,17 @@ export class KafkaClient extends Client {
         return topic.replace(illegalChars, "-");
     }
 
+    /**
+     * Decomposes a topic name to extract it's global topic name.

```

Comment:
Maybe add a line like

"For example, if the topic is named "global:subdomain", the global topic is called "global".